assets/controllers/password_strength_controller.js

```diff
@@ -3,10 +3,11 @@ import { getValidCriteria, isPasswordStrongEnough } from './helpers/password_str
 
 export default class extends Controller {
   static targets = ['input', 'badge', 'widget', 'validText', 'invalidText'];
+  static values = { passwordLength: String };
 
-  check() {
+  check () {
     const value = this.inputTarget.value;
-    const validCriteria = getValidCriteria(value);
+    const validCriteria = getValidCriteria(value, this.passwordLengthValue);
```

Comment:
Ici on peut faire un `connect() { this.helper = new PasswordLenghtHelper(this.passwordLengthValue) }`